File: src/composite/Composite.java
package composite;

import java.util.ArrayList;
import java.util.List;

public class Composite extends Component {
    List<String> l = new ArrayList<String>();

    public void addChild(String comp){
        l.add(comp);
        System.out.println("Number of components: " + l.size());
    }

    @Override
    public void operation() {
        System.out.println("Composite");
    }
}

File: Scripts/composite/Composite.php
<?php
require_once('Component.php');

class Composite extends Component {
	private $components = array();
	
	public function addChild($comp){
		$this->components[] = $comp;
		echo "Number of components: " . count($this->components) . "<br/>";
	}
	
	public function operation(){
		echo "Composite<br/>";
	}
}
?>